ui/main/src/app/modules/admin/components/ngtable/groups/of-groups-table.component.ts

```diff
@@ -0,0 +1,104 @@
+
+@Component({
+  selector: 'of-groups-table',
+  templateUrl: '../oftable/oftable.component.html'
+})
+export class OfGroupsTableComponent extends OfTableComponent implements OnInit {
+
+  modalComponent = EditEntityGroupModalComponent;
+  typeModal = 'group';
+  constructor(
+    protected crudService: GroupsService,
+    protected modalService: NgbModal,
+    protected dataService: DataTableShareService,
+    protected translate: TranslateService,
+    private confirmationDialogService: ConfirmationDialogService,
+    protected store: Store<AppState>
+  ) {
+    super(modalService, dataService, store);
+    this.crudService = crudService;
+    this.config.sorting = { columns: this.columns };
+    this.getLocale().subscribe(locale => {
+      this.translate.use(locale);
+      this.translate.get(['admin.input.id.label', 'admin.input.name.label', 'admin.input.description',
+        'admin.input.user.edit', 'admin.input.user.delete', 'admin.input.user.filter', 'admin.input.user.filter',
+        'admin.pagination.firstText', 'admin.pagination.lastText', 'admin.pagination.nextText', 'admin.pagination.prevText'
+        , 'admin.input.group.add', 'admin.input.lines', 'admin.input.user.filter', 'admin.input.user.filterAll'])
+        .subscribe(translations => {
+          this.columns = [
+            {
+              title: translations['admin.input.id.label'],
+              name: 'id',
+              filtering: {
+                filterString: '', placeholder: translations['admin.input.user.filter'] +
+                  translations['admin.input.id.label']
+              },
+            },
+            {
+              title: translations['admin.input.name.label'],
+              name: 'name',
+              filtering: {
+                filterString: '', placeholder: translations['admin.input.user.filter'] +
+                  translations['admin.input.name.label']
+              },
+            },
+            {
+              title: translations['admin.input.description'],
+              name: 'description',
+              filtering: {
+                filterString: '', placeholder: translations['admin.input.user.filter'] +
+                  translations['admin.input.description']
+              }
+            },
+            {
+              title: translations['admin.input.user.edit'],
+              name: 'edit'
+            },
+            {
+              title: translations['admin.input.user.delete'],
+              name: 'delete'
+            }
+          ];
+          this.firstText = translations['admin.pagination.firstText'];
+          this.lastText = translations['admin.pagination.lastText'];
+          this.nextText = translations['admin.pagination.nextText'];
+          this.prevText = translations['admin.pagination.prevText'];
+          this.addLabel = translations['admin.input.group.add'];
+          this.lineLabel = translations['admin.input.lines'];
+          this.filterAll = translations['admin.input.user.filterAll'];
+
```

Comment:
Duplication of code with other classes . 
See how to set stuff in the parent class  and create a method instead of repeating  this blocks : 

` title: translations['admin.input.description'],
              name: 'description',
              filtering: {
                filterString: '', placeholder: translations['admin.input.user.filter'] +
                  translations['admin.input.description']
              }`